### PR TITLE
Temporary fix for missing .ivy2 directory

### PR DESCRIPTION
**Pull request by rawyler**
Somehow the .ivy2 directory went missing since the 1.5.0 build.

Also added git since it's not part of the oraclelinux base image anymore.

### CODE REVIEW

File: oracle/Dockerfile [hunk 1]
```diff
@@ -18,6 +18,9 @@ ENV USER_ID ${USER_ID:-1001}
 ARG GROUP_ID 
 ENV GROUP_ID ${GROUP_ID:-1001}
 
+# Install git because it's very useful to have that should the version be extracted from git
```
Comment (francisdb) [inline oracle/Dockerfile:21]: Would be better have this in a different pr, and applied to all images

File: oracle/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 # Pull base image
 ARG BASE_IMAGE_TAG
-FROM openjdk:${BASE_IMAGE_TAG:-11.0.10-jdk-oraclelinux8}
+FROM openjdk:${BASE_IMAGE_TAG:-16-jdk-oraclelinux8}
```
Comment (francisdb) [inline oracle/Dockerfile:9]: Is this required? Maybe you can just add 16-jdk-oraclelinux8 to the build matrix? Preferable also in a different pr

Comment (rawyler) [reply oracle/Dockerfile:9]: That's not required, you're right. Then the previous line with the installation of git wouldn't be required too.

File: oracle/Dockerfile (hunk 1 above)
Comment (rawyler) [reply oracle/Dockerfile]: I would have to check all the listed base images in the workflows for git installations

File: oracle/Dockerfile [hunk 3]
```diff
@@ -70,6 +70,8 @@ RUN \
 # This allows users of this container to choose, whether they want to run the container as sbtuser (non-root) or as root
 USER root
 RUN \
+  mkdir /home/sbtuser/.ivy2 && \
```
Comment (francisdb) [inline oracle/Dockerfile:73]: I guess the other images will have the same problem? Can you also add these commands in there?

**Review by francisdb — CHANGES_REQUESTED** [review]

**Review by francisdb — APPROVED** [review]

**Comment by francisdb** [comment]
Thanks!
